From ccc4162ee8ca16ce560b1efdc90cafa67cbc1415 Mon Sep 17 00:00:00 2001
From: Dan Williams <dcbw@redhat.com>
Date: Tue, 12 May 2015 15:43:18 -0500
Subject: [PATCH] fedora: don't try to use biosdevname if it's not installed

If biosdevname isn't installed it doesn't make sense to try using it
for persistent device names.  Just treat lack of biosdevname as
virtual networking.
---
 plugins/guests/fedora/cap/configure_networks.rb | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/plugins/guests/fedora/cap/configure_networks.rb b/plugins/guests/fedora/cap/configure_networks.rb
index 2f67099..8e071b5 100644
--- a/plugins/guests/fedora/cap/configure_networks.rb
+++ b/plugins/guests/fedora/cap/configure_networks.rb
@@ -17,7 +17,7 @@ def self.configure_networks(machine, networks)
           virtual = false
           interface_names = Array.new
           machine.communicate.sudo("/usr/sbin/biosdevname; echo $?") do |_, result|
-            virtual = true if result.chomp == '4'
+            virtual = true if ['4', '127'].include? result.chomp
           end
 
           if virtual

